contracts/LoanManager.sol

```diff
@@ -274,11 +275,12 @@ contract LoanManager is Restricted, TokenReceiver {
         (loanAmount, interestAmount) = calculateLoanValues(product, loan.repaymentAmount);
         uint disbursementTime = loan.maturity - product.term;
 
-        LoanState loanState =
-                loan.state == LoanState.Open && now >= loan.maturity ? LoanState.Defaulted : loan.state;
+        // Add extra calculated data for convenience: marginCallRate, isDefaulted
+        uint marginCallRate = calculateMarginCallRate(product.minCollateralRatio, loan.repaymentAmount, loan.collateralAmount);
 
         result = [loanId, loan.collateralAmount, loan.repaymentAmount, uint(loan.borrower),
-                loan.productId, uint(loanState), loan.maturity, disbursementTime, loanAmount, interestAmount];
+            loan.productId, uint(loan.state), loan.maturity, disbursementTime, loanAmount, interestAmount,
+            marginCallRate, isDefaulted(loan, currentRate) ? 1 : 0];
```

Comment:
I think we shouldn't mix `isDefaulted` and `isUnderMargin` those are two different things. 
I.e. If a loan is under margin then it can still recover if someone tops up collateral. While a defaulted loan (which might be collected or not) can never recover, it can only be collected.

Reply:
Suggestion: call this property (and the method) `canCollect` or `collectable`

Reply:
I've renamed it.

Reply:
Would an additional, calculated prop such as `isExpired` or `isDefaulted` still make sense? 
It could be calculated on the client too  but a bit wordy, something like: 
`maturity < now && (status == Open || status == Collected))`
Also client side calculation would require "blocktime" to be "in sync" with the client clock. 
I'm just thinking to settle with the interface early to avoid multiple changes of `@augmint/js` interface changes.